packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -0,0 +1,95 @@
+/** @jsx withSlots */
+import { useRef, useEffect, useMemo } from 'react';
+import { Animated, Easing } from 'react-native';
+import { Svg, Path } from 'react-native-svg';
+import { compose, mergeProps, withSlots, UseSlots, buildUseStyling } from '@fluentui-react-native/framework';
+import { activityIndicatorName, ActivityIndicatorProps, ActivityIndicatorType } from './ActivityIndicator.types';
+import { diameterSizeMap, lineThicknessSizeMap, stylingSettings } from './ActivityIndicator.styling';
+
+// Can also just put this within the compose, would that be better?
```

Comment:
I think this is fine here. I have a feeling if you put it inside the `render` of compose, every time we call render, it would recreate this function.